CONTRIBUTORS.md

```diff
@@ -2002,6 +2004,8 @@
 
 - [@incanlove](https://github.com/incanlove)
 
+-[@incanlove](https://github.com/incanlove)
```

Comment:
This is a (badly formatted) duplicate of the entry above. Can you please remove this?

Reply:
Issue has been resolved and I will re-submit pull request.